### PR TITLE
made Game class abstract and created playerGame and dealerGame

### PR DESCRIPTION
**Pull request by charan96**

### CODE REVIEW

File: src/main/java/models/dealerGame.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package models;
+
+/**
+ * Created by ramcharan on 3/10/16.
+ */
+public class dealerGame extends Game
```
Comment (schultsa) [inline src/main/java/models/dealerGame.java:6]: Will start implementing split.

File: src/main/java/models/Card.java [hunk 2]
```diff
@@ -8,11 +8,11 @@
  */
 public class Card
 {
-    public final int value;
```
Comment (dennislee15) [inline src/main/java/models/Card.java:11]: why did you change from int to string?

Comment (charan96) [inline src/main/java/models/Card.java:11]: We have an A for Ace because Ace can either be 11 or 1. We change it to 11 or 1 based on the total score of the player or dealer at runtime.
